docs/api_reference.md

```
@@ -0,0 +1,514 @@
+- [Module attributes](#module-attributes)
+- [Signal generators](#signal-generators)
+- [Window functions](#window-functions)
+- [Quantization](#quantization)
+- [Spectral analysis](#spectral-analysis)
+- [IIR filter design — classical families](#iir-filter-design--classical-families)
+- [IIR filter design — RBJ biquads](#iir-filter-design--rbj-biquads)
+- [FIR filter design](#fir-filter-design)
+- [Image — generators](#image--generators)
+- [Image — processing](#image--processing)
+- [Image — morphology](#image--morphology)
+- [Image — file I/O](#image--file-i/o)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)